app.js

```diff
@@ -3,22 +3,37 @@ const api = require('./api')
 const middleware = require('./middleware')
 const bodyParser = require('body-parser')
 
-
 // Set the port
 const port = process.env.PORT || 3000
+
 // Boot the app
 const app = express()
+
 // Register the public directory
 app.use(express.static(__dirname + '/public'));
+
 // register the routes
 app.use(bodyParser.json())
 app.use(middleware.cors)
+
+// Register root route
 app.get('/', api.handleRoot)
+
+// Register Products routes
 app.get('/products', api.listProducts)
 app.get('/products/:id', api.getProduct)
 app.put('/products/:id', api.editProduct)
 app.delete('/products/:id', api.deleteProduct)
 app.post('/products', api.createProduct)
+
 // Boot the server
 app.listen(port, () => console.log(`Server listening on port ${port}`))
 
+
+
+// app.js
+
+// ...
+
+app.get('/orders', api.listOrders)
+app.get('/orders/', api.createOrder)
```

Comment:
```suggestion
app.post('/orders/', api.createOrder)
```

We're also missing the update and delete routes for the orders